src/cpp.cc

```diff
@@ -2518,7 +2699,10 @@ void Printer::Cpp::makefile(const Options &opts) {
   stream << "DEPS = $(CXXFILES:.cc=.d)" << endl
     << "OFILES = $(CXXFILES:.cc=.o) string.o" << endl << endl;
   stream << opts.class_name << " : $(OFILES)" << endl
-    << "\t$(CXX) -o $@ $^  $(LDFLAGS) $(LDLIBS)";
+      << "\t$(CXX) -o $@ $^  $(LDFLAGS) $(LDLIBS)";
+  if ((opts.checkpointing)) {
```

Comment:
why `((` and not `(` ?

Reply:
Don't know how that happened. I removed the extra braces.